test/blocks/article-header/article-header.test.js

```diff
@@ -0,0 +1,85 @@
+import { readFile } from '@web/test-runner-commands';
+import { expect } from '@esm-bundle/chai';
+import sinon from 'sinon';
+
+import { setConfig, getConfig } from '../../../libs/utils/utils.js';
+import { delay, waitForElement } from '../../helpers/waitfor.js';
+
+const locales = { '': { ietf: 'en-US', tk: 'hah7vzn.css' } };
+const conf = { locales, miloLibs: 'http://localhost:2000/libs' };
+setConfig(conf);
+const config = getConfig();
+
+document.body.innerHTML = await readFile({ path: './mocks/body.html' });
+const { default: init } = await import('../../../libs/blocks/article-header/article-header.js');
+
+const invalidDoc = await readFile({ path: './mocks/body-invalid.html' });
+
+describe('article header', () => {
+  const block = document.body.querySelector('.article-header');
+
+  it('creates article header block', async () => {
+    config.locale.contentRoot = '/test/blocks/article-header/mocks';
+
+    await init(block);
+    expect(document.body.querySelector('.article-category')).to.be.exist;
```

Comment:
can remove `be` from the assertion: `expect(document.body.querySelector('.article-category')).to.exist`;

this occurs elsewhere where you could remove it.